src/api_manager/auth/lib/auth_jwt_validator.cc

```diff
@@ -569,10 +569,6 @@ grpc_jwt_verifier_status JwtValidatorImpl::ExtractAndVerifyJwkKeys(
   // JWK format from https://tools.ietf.org/html/rfc7518#section-6.
   for (jkey = jwk_keys->child; jkey != nullptr; jkey = jkey->next) {
     if (jkey->type != GRPC_JSON_OBJECT) continue;
```

Comment:
Will there be cases where "alg" is in some of sub-keys?

Reply:
We could just ignore that, the spec says "alg" is optional. the "kid" should be strong enough for matching.